crates/store/re_types/src/blueprint/components/background_kind.rs

```diff
@@ -20,20 +20,21 @@ use ::re_types_core::{DeserializationError, DeserializationResult};
 
 /// **Component**: The type of the background in a view.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, Default)]
+#[repr(u8)]
 pub enum BackgroundKind {
     /// A dark gradient.
     ///
     /// In 3D views it changes depending on the direction of the view.
     #[default]
-    GradientDark = 1,
+    GradientDark = 0u8,
```

Comment:
nit, but we can make the generated code more readable if you quote a [`Literal::u8_unsuffixed`](https://docs.rs/proc-macro2/latest/proc_macro2/struct.Literal.html#method.u8_unsuffixed) instead

```suggestion
    GradientDark = 0,
```